src/types/custom.ts

```diff
@@ -0,0 +1,20 @@
+import { Runtype, Rt, Static, create, validationError } from '../runtype'
+import { String } from './string'
+
+export interface Custom<A extends Rt, T extends string, K> extends Runtype<Static<A>> {
+  tag: T,
```

Comment:
`tag` should just be `'custom'`, because it's meant to be used when doing case analysis across the closed set of built-in runtypes. I like the idea of having a custom type as an escape hatch out of the closed world, but doesn't the `args: K` field already give you the power to stuff in whatever information you want? For example, you could have `args: { tag: 'my-custom-type', x: number, y: boolean}`.